### ISSUE TITLE
0.3.5 and 0.4.* are incompatible

### ISSUE THREAD

**Issue by freiit** [okgrow/meteor-persistent-session#45]
If you upgrade your project, the values stored in a browser cache might not be the same anymore.

*# meteor add u2622:persistent-session@=0.3.5
**$ Session.setPersistent('x', 5)
**$ Session.setPersistent('y', null)
**$ Session.get('x')
5
**$ Session.get('y')
null

*# meteor add u2622:persistent-session@=0.4.1
** $ Session.get('x')
"5"
**$ Session.get('y')
"null"

**Comment by godblessmensah** [comment]
Am facing this same issue. Can something be done about it?
This should really be fixed for us please.

**Comment by aramk** [comment]
:+1:

**Comment by aramk** [comment]
Perhaps a migration would be good - if they were stored without quotes before you use regex to determine if they should be strings?

**Comment by deanius** [comment]
A migration is included in PR #47  - it's just not gone live yet- I gave it a +1  - can anyone else help by checking it out ?

**Comment by aramk** [comment]
@deanius I tested by adding values on 0.3.5 then moving to 0.4.1 then to [5b53df1]. I tested these values and it looks like it works after breaking in 0.4.1:

```
Session.get('a')
> "foo"
Session.get('b')
> 123
Session.get('c')
> true
Session.get('d')
> false
Session.get('e')
> Object {abc: 123}
```

Thanks for that, guys!

**Comment by deanius** [comment]
@rgould  - looks good to merge

**Comment by rgould** [comment]
Fixed by #47
